aws_lambda_powertools/utilities/parameters.py

```diff
@@ -0,0 +1,63 @@
+"""
+Parameter retrieval and caching utility
+"""
+
+
+from collections import namedtuple
+from datetime import datetime, timedelta
+
+import boto3
+
+DEFAULT_MAX_AGE = 5
```

Comment:
Could this be renamed to DEFAULT_MAX_AGE_SECS or something to reflect the time unit?